thorlcr/activities/fetch/thfetchslave.cpp

```diff
@@ -57,7 +58,6 @@ class CFetchStream : public IRowStream, implements IStopInput, implements IFetch
     Linked<IExpander> eexp;
 
     FPosTableEntryIFileIO *fPosMultiPartTable;
-    unsigned tableSize;
```

Comment:
NB: not directly related to the issue, but noticed whilst reviewing changed.

unused member var.